Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,232 @@
+***
```

Comment:
The whole section needs to explain what is being explained in this wiki page which is: using the library from a client perspective to authenticate using http-signature spec to a server in order to access a protected resource.

This requires:
  1. making a public-private key pair
  2. storing the private key locally for future re-use
  3. publishing the public key on some web server (we will publish it on the local rww-play server)
  4. changing the acl of a resource to restrict access to the person owning that key 
  5. then accessing that resource with that key using http-signature